test/tests/menubar-2.js

```diff
@@ -0,0 +1,1086 @@
+'use strict';
+
+const { ariaTest } = require('..');
+const { By, Key } = require('selenium-webdriver');
+const confirmAttributeValue = require('../util/confirmAttributeValue');
+const confirmRovingTabindex = require('../util/confirmRovingTabindex');
+
+const exampleFile = 'menubar/menubar-2/menubar-2.html';
+
+let ex = {
```

Comment:
The properties of `ex` are never mutated beyond this point. This can be `const`.